tests/unit/backend/layers/business/test_business.py

```diff
@@ -1398,18 +1398,34 @@ def test_delete_collection_version_ok(self):
         self.assertEqual(version.version_id, published_collection.version_id)
         self.assertNotEqual(version.version_id, new_version.version_id)
 
-    def test_tombstone_collection_ok(self):
+    def test_tombstone_then_resurrect_collection_ok(self):
         """
-        A collection can be marked as tombstoned using 'tombstone_collection'
+        A Collection can be tombstoned and then resurrected/untombstoned
```

Comment:
resurrected or untombstoned. I prefer **restore** but resurrect also works

Reply:
I prefer "resurrect" because it is more directly associated with "tombstone" as being its opposite. I think "restore" makes more sense in the context of restoring deleted s3 assets.